ui/app/AppLayouts/Wallet/services/dapps/helpers.js

```diff
@@ -69,30 +69,30 @@ function buildSupportedNamespaces(chainIds, addresses, methods) {
 }
 
 function validURI(uri) {
-    var regex = /^wc:[0-9a-fA-F-]*@([1-9][0-9]*)(\?([a-zA-Z-]+=[^&]+)(&[a-zA-Z-]+=[^&]+)*)?$/
+    const regex = /^wc:[0-9a-fA-F-]*@([1-9][0-9]*)(\?([a-zA-Z-]+=[^&]+)(&[a-zA-Z-]+=[^&]+)*)?$/
     return regex.test(uri)
 }
 
-function containsOnlyEmoji(uri) {
-    var emojiRegex = new RegExp("[\\u203C-\\u3299\\u1F000-\\u1F644]");
-    return !emojiRegex.test(uri);
+function containsEmoji(uri) {
+    const emojiRegex = /^.*[\u{1F300}-\u{1F5FF}\u{1F600}-\u{1F64F}\u{1F680}-\u{1F6FF}\u{1F700}-\u{1F77F}\u{1F780}-\u{1F7FF}\u{1F800}-\u{1F8FF}\u{1F900}-\u{1F9FF}\u{1FA00}-\u{1FA6F}\u{1FA70}-\u{1FAFF}\u{1F1E6}-\u{1F1FF}]+/u;
```

Comment:
We have 2 similar regexes already under `Constants.regularExpressions`

Reply:
Thanks, I remove it and replaced with `Constants.regularExpressions.emoji.test(uri)`